lib/exercism/track_stream.rb

```diff
@@ -143,7 +144,8 @@ def exercises_sql
         ex.last_activity_at,
         ex.iteration_count,
         u.username,
-        u.avatar_url
+        u.avatar_url,
+        ex.request_help
```

Comment:
It would probably be more consistent with the existing code to group this with the other `ex` fields, rather than adding it to the bottom.